src/vl-map-layer.js

```diff
@@ -109,6 +115,10 @@ export class VlMapLayer extends vlElement(HTMLElement) {
     return this.getAttribute('max-resolution') || Infinity;
   }
 
+  get _visible() {
+    return this.getAttribute('hidden') == undefined;
```

Comment:
=== ?

Reply:
In deze zou ik dat niet doen, want null zorgt nu ook voor true.